config/environments/production.rb

```diff
@@ -26,8 +26,6 @@
 
   # Compress JavaScripts.
   config.assets.js_compressor = :uglifier
-  # Compress CSS using a preprocessor.
-  # config.assets.css_compressor = :sass
 
   # Do not add source map line comments into css files
   config.sass.line_comments = false
```

Comment:
It looks like `config.sass.line_comments` was added twice (probably because of a Rails update). Could you please remove it and the comment on line 30.